package.json

```diff
@@ -1,14 +1,21 @@
 {
-  "name": "api-client-js",
+  "name": "@accentor/api-client-js",
   "version": "0.1.0",
   "description": "A javascript client for the Accentor API",
-  "main": "src/index.js",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "repository": "github:accentor/api-client-js",
   "scripts": {
-    "lint": "eslint . --ext .js --fix"
+    "lint": "eslint . --ext .js --fix",
+    "build": "node ./esbuild.js"
   },
   "author": "Robbe Van Petegem <robbe@vanpetegem.me>",
```

Comment:
When publishing this to the accentor organization, wouldn't it be better to make the author "Team Accentor", like in the license? (Not trying to take credit for something I didn't do a whole lot on, it just seems more consistent.)

Reply:
true, it's clearer if the org is the author.
I've added Team Accentor as author and added a contributor field for individual peopel (feel free to add yourself)